lib/main.dart

```diff
@@ -3,124 +3,114 @@ import 'package:english_words/english_words.dart';
 
 void main() => runApp(new MyApp());
 
+const minPlayers = 5;
+const maxPlayers = 10;
+
 class MyApp extends StatelessWidget {
+
   @override
   Widget build(BuildContext context) {
+
     return new MaterialApp(
-      title: 'Welcome to Flutter',
+      title: 'Heist',
       theme: new ThemeData(
         primaryColor: Colors.deepOrange,
       ),
-      home: new RandomWords()
+      home: new Scaffold(
+        appBar: new AppBar(
+          title: new Text("Heist"),
+        ),
+        body: new HomePage()
+      )
     );
   }
 }
 
-class RandomWords extends StatefulWidget {
+class HomePage extends StatefulWidget {
   @override
-  createState() => new RandomWordsState();
+  State<StatefulWidget> createState() {
+    return new HomePageState();
+  }
 }
 
-class RandomWordsState extends State<RandomWords> {
-
-  final _suggestions = <WordPair>[];
-  final _saved = new Set<WordPair>();
-  final _biggerFont = const TextStyle(fontSize: 18.0);
-
-  void _pushSaved() {
-    Navigator.of(context).push(
-      new MaterialPageRoute(
-        builder: (context) {
-          final tiles = _saved.map(
-                  (pair) {
-              return new ListTile(
-                title: new Text(
-                  pair.asPascalCase,
-                  style: _biggerFont,
-                ),
-              );
-            },
-          );
-          final divided = ListTile
-              .divideTiles(
-            context: context,
-            tiles: tiles,
-          ).toList();
-
-          return new Scaffold(
-            appBar: new AppBar(
-              title: new Text('Saved Suggestions'),
-            ),
-            body: new ListView(children: divided),
-          );
-        },
-      ),
-    );
-  }
+class HomePageState extends State<HomePage> {
+
+  var _numPlayers = minPlayers;
 
   @override
   Widget build(BuildContext context) {
-    return new Scaffold (
-      appBar: new AppBar(
-        title: new Text('Startup Name Generator'),
-        actions: <Widget>[
-          new IconButton(icon: new Icon(Icons.list), onPressed: _pushSaved),
-        ],
+
+    Column buildArrowColumn(IconData icon, Function onPressed) {
+      Color color = Theme.of(context).primaryColor;
+      return new Column(
+          children: [
+            new IconButton(
+              iconSize: 64.0,
+              onPressed: onPressed,
+              icon: new Icon(icon, color: color)
+            )
+          ],
+      );
+    }
+
+    Widget numPlayersText = new Container(
+      padding: const EdgeInsets.all(32.0),
+      child: new Text(
+        _numPlayers.toString(),
+        style: new TextStyle(
```

Comment:
The other TextStyles have the value in the same line and they're const (lines 70 and 101). This one should be consistent, unless there's a reason for it to be different that I'm missing.

Reply:
Yes, well spotted, it can be const, it's just the Text object above which obviously can't.